hazelcast/src/main/java/com/hazelcast/config/QueueConfig.java

```diff
@@ -323,6 +324,15 @@ public QueueConfig setMergePolicyConfig(MergePolicyConfig mergePolicyConfig) {
         return this;
     }
 
+    public Comparator getComparator() {
```

Comment:
Can you add some javadoc for the getter and setter? They can be almost identical except for the difference where you say that one returns while the other one sets, like [here](https://github.com/hazelcast/hazelcast/blob/97ff57ba92dc1fbfa77720885e7db40757999080/hazelcast/src/main/java/com/hazelcast/config/WanBatchPublisherConfig.java#L138-L167).